livekit-plugins/livekit-plugins-gladia/README.md

```diff
@@ -0,0 +1,76 @@
+# LiveKit Plugins Gladia
+
+Agent Framework plugin for speech-to-text with [Gladia](https://gladia.io/)'s API. Currently supports streaming speech-to-text with optional translation.
+
+## Installation
+
+```bash
+pip install livekit-plugins-gladia
```

Comment:
you should add `gladia = ["livekit-plugins-gladia>=1.0.0.rc4"]` to the main pyproject.toml  as well so this can be `pip install livekit-agents[gladia]`

Reply:
Done!